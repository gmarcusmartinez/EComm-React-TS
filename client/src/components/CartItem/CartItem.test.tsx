import { shallow } from 'enzyme';
import React from 'react';
import CardItem from '.';

const mockItem = {
  id: 1,
  quantity: 1,
  name: 'Mock Item',
  price: 9.99,
  imageUrl: 'image.com',
};

describe('CartItem', () => {
  test(' component renders', () => {
    expect(shallow(<CardItem item={mockItem} />)).toMatchSnapshot();
  });
});
