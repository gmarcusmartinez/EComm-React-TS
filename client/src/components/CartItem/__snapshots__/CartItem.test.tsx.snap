// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CartItem  component renders 1`] = `
<div
  className="cart-item"
>
  <img
    alt="item"
    src="image.com"
  />
  <div
    className="item-details"
  >
    <span
      className="name"
    >
      Mock Item
    </span>
    <span
      className="price"
    >
      1
       x $
      9.99
    </span>
  </div>
</div>
`;
